rdfunit-core/src/main/java/org/aksw/rdfunit/io/writer/RDFXMLResultsStatusWriter.java

```diff
@@ -0,0 +1,122 @@
+package org.aksw.rdfunit.io.writer;
+
+import org.aksw.jena_sparql_api.core.QueryExecutionFactory;
+import org.aksw.rdfunit.services.PrefixNSService;
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QuerySolution;
+import org.apache.jena.query.ResultSet;
+
+import java.io.OutputStream;
+
+/**
+ * <p>RDFHTMLResultsStatusWriter class.</p>
+ *
+ * @author Martin Bruemmer
+ *         Description
+ * @since 4/23/14 8:55 AM
+ * @version $Id: $Id
+ */
+public class RDFXMLResultsStatusWriter extends RDFXMLResultsWriter {
+
+    /**
+     * <p>Constructor for RDFHTMLResultsStatusWriter.</p>
+     *
+     * @param filename a {@link java.lang.String} object.
+     */
+    public RDFXMLResultsStatusWriter(String filename) {
```

Comment:
didn't have time to change the HTML results but better init it with e TestExecution and access the results with java objects directly